MaxiOps/CCTP_Bridge/receive/claimArbiUSDC.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": "1.0",
+  "chainId": "42161",
+  "createdAt": 1715193245588,
+  "meta": {
+    "name": "Transactions Batch",
+    "description": "",
+    "txBuilderVersion": "1.16.5",
+    "createdFromSafeAddress": "0xc38c5f97B34E175FFd35407fc91a937300E33860",
+    "createdFromOwnerAddress": "",
+    "checksum": "0x383bbde1a0fa18089fef4d0f07fca15a785b14563241a3e34530459e2fdddc97"
+  },
+  "transactions": [
+    {
+      "to": "0xC30362313FBBA5cf9163F0bb16a0e01f01A896ca",
```

Comment:
Pretty sure the receiver is the same on all chains.  Not quite sure where to check it, I just looked at arbtirum last time.  Probably in the docs somewhere if tis not the same, Will try on base.